packages/site-demo/content/product/components/popover/index.mdx

```diff
@@ -1,36 +1,44 @@
 # Popover
 
-## Default behavior
+**Popovers display pieces of UI on top of others. They are positioned from anchor elements.**
 
-Popover can be used to display elements above the rest of the UI.
+## Default
 
 <DemoBlock demo="default" />
 
-## Offset
+## Positions
 
-Popover can be offseted from its anchor point.
+### Directed
 
-<DemoBlock demo="offset" />
+Popovers can be placed to 4 positions :
+_TOP_, _RIGHT_, _BOTTOM_, _LEFT_
 
-## Control the popper visibility
+<DemoBlock demo="positions" />
 
-<DemoBlock demo="visibility" />
+### Auto
 
-## Using anchor point
+Popovers can also be placed automatically where space is available. They will be placed at the bottom of anchors by default.
```

Comment:
```suggestion
Popovers can also be placed automatically where space is available. They are be placed at the bottom of anchors by default.
```
It's always better to avoid using the future tense.